src/client/pages/signIn/SignIn.jsx

```diff
@@ -10,45 +15,71 @@ const emptyUser = {
 const SignIn = () => {
     const [loginDetails, setLoginDetails] = useState(emptyUser);
 
+    const postLogin = async (url, loginDetails) => {
+        const res = await fetch(url, {
+            method: 'POST',
+            headers: {
+                'Content-Type': 'application/json',
+                Authorization: localStorage.getItem(),
```

Comment:
Hmmmm - you should probably `getItem` first, then check that you have successfully got and only `fetch` if so - it would make the code much more robust...

Reply:
I quite honestly have no idea what I am doing